migrator/migrations/m_207_to_m_208_initialize_nvd_cvss/migration_impl.go

```diff
@@ -0,0 +1,32 @@
+package m207tom208
+
+import (
+	newSchema "github.com/stackrox/rox/migrator/migrations/m_207_to_m_208_initialize_nvd_cvss/schema"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+)
+
+// If so increment the `MinimumSupportedDBVersionSeqNum` to the `CurrentDBVersionSeqNum` of the release immediately
```

Comment:
Can this comment block be removed or does the check-generated test fail if removed?

Reply:
removed